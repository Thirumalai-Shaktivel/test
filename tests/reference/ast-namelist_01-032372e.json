{
    "basename": "ast-namelist_01-032372e",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/namelist_01.f90",
    "infile_hash": "82b81ebf89597c4cee2bae25cbe602a58d7d628cf9dec14337630861",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-namelist_01-032372e.stdout",
    "stdout_hash": "e004b75038babb9a2afe8c73b09d2d9f1031c54dc2dbc45d20f9b208",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}