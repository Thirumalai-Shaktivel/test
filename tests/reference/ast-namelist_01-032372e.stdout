(TranslationUnit [(Program namelist1 (TriviaNode [] []) [] [(ImplicitNone [] (TriviaNode [] []))] [(Declaration (AttrType TypeInteger [] () None) [] [(N [] [] () None ())] (TriviaNode [] [])) (Declaration (AttrType TypeReal [] () None) [] [(t [] [] () None ()) (x [] [] () None ()) (Ecut [] [] () None ())] (TriviaNode [] [])) (Declaration () [(AttrNamelist domain)] [(N [] [] () None ()) (t [] [] () None ()) (x [] [] () None ()) (Ecut [] [] () None ())] (TriviaNode [] []))] [] [])])
